docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-en.md

```diff
@@ -0,0 +1,141 @@
+
+5. Start Dashboard.
+
+   You can use the following command to start the Dashboard with one click.
+   ```bash
+   $ bash ./scripts/start.sh
+   ```
+   Or execute the following commands to start prometheus, webserver, exporter and gateway services to start Dashboard.
+   
+   ```bash
+   $ cd scripts
+   $ sudo ./dashboard.server start prometheus # Start prometheus service
+   $ sudo ./dashboard.server start webserver # Start webserver service
+   $ sudo ./dashboard.server start exporter # Start exporter service
+   $ sudo ./dashboard.server start gateway # Start gateway service
+   ```
+
+## Manage Dashboard Service
+
+You can use the `dashboard.service` script to start, stop, and check the Dashboard services.
+
+### Syntax
+
+```bash
+$ sudo <dashboard_path>/dashboard/scripts/dashboard.service
+[-v] [-h]
+<start|stop|status>  <prometheus|webserver|exporter|gateway|all>
+```
+
+| Parameter                  | Description       |
+| :------------------------- | :------------------- |
+| `dashboard_path` | Dashboard installation path.  |
+| `-v`                       | Display detailed debugging information.   |
+| `-h`                       | Display help information.        |
+| `start`                    | Start the target services.       |
+| `stop`                     | Stop the target services.           |
+| `status`                   | Check the status of the target services.       |
+| `prometheus`               | Set the prometheus Service as the target service. |
+| `webserver`                | Set the webserver Service as the target service.  |
+| `exporter`                 | Set the exporter Service as the target service.   |
+| `gateway`                  | Set the gateway Service as the target service.    |
+| `all`                      | Set all the Dashboard services as the target services.       |
+
+### Examples
+
+Dashboard is installed in the current directory, and you can use the following commands to manage services.
+```bash
+$ sudo /dashboard/scripts/dashboard.service start all #start all Dashboard service 
+$ sudo /dashboard/scripts/dashboard.service stop all #stop all Dashboard service 
+$ sudo /dashboard/scripts/dashboard.service status all #check all Dashboard service 
+```
+
+## Next to do
+
+After Dashboard is successfully started, you can enter `http://<ip_address>:7005` in the address bar of a browser.
+
+If the following login interface is shown in the browser, then you have successfully deployed and started Dashboard. You can log into the GOD user of Dashboard with the default username `nebula` and password `nebula`. You can modify the password in [System Settings](../nebula-dashboard-ent/6.system-settings.md). And you also can use the GOD account to create a new account with the permission ADMIN on the [Account Management](../nebula-dashboard-ent/5.account-management.md) to log into the Dashboard.
```

Comment:
You can log into the GOD user of Dashboard with the default username `nebula` and password `nebula`.

--->You can log into the Dashboard as a GOD user with the xxx